test/error_handling/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+
+macro(run_test case)
+get_filename_component(test_name ${case} NAME_WE)
+set(test_comment "! ")
+add_test(NAME ${test_name}.error_handling
+  COMMAND $<TARGET_FILE:CSE> -x${test_comment} -b -t1 "${test_name}"
+  WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}"
+)
+set_tests_properties(${test_name}.error_handling PROPERTIES WILL_FAIL TRUE)
+endmacro()
+
+macro(error_checking case)
```

Comment:
The convention for the other macros is "xxxx_test". These are really error regression tests, right? You are checking to see if there has been a regression in the error content?